### PR TITLE
added proposion

### PR DESCRIPTION
**Pull request by Sskeeper**
added proosion about share work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ Just studing repository
 now I'm doing my course by guide
 
 Second attempt
+
+# lets work together
```
Comment (Hardbow) [inline README.md]: Maybe it's need " ' "?

**Review by Hardbow — CHANGES_REQUESTED** [review]
It's nesessary to make a correcting

File: README.md (hunk 1 above)
Comment (Hardbow) [inline README.md]: have you received my messenge?

**Comment by Sskeeper** [comment]
it's done, apostrophe is added

**Review by Hardbow — APPROVED** [review]

**Pull request merged**